skiko/src/commonMain/kotlin/org/jetbrains/skia/paragraph/StrutStyle.kt

```diff
@@ -36,14 +38,18 @@ class StrutStyle internal constructor(ptr: NativePointer) : Managed(ptr, _Finali
     val fontFamilies: Array<String>
         get() = try {
             Stats.onNativeCall()
-            _nGetFontFamilies(_ptr)
+            arrayDecoderScope({ArrayDecoder(_nGetFontFamilies(_ptr), ManagedString_nGetFinalizer())}) { arrayDecoder ->
```

Comment:
This becomes an idiom, so maybe we should make a helper from it?

Reply:
Let's wait till we get even more usages of ArrayDecoder and see what can be generalized - I mean your suggestion with moving release logic entirely to the backend has a lot of sense and we'll see how it goes )))

Reply:
@Schahen I mean we can move these 3 lines into a helper function (like `fromInteropStringArray`) and use it here and in the other places where we a receiving strings as an array. Also there is generic implementation of `fromInterop` for `ArrayInteropDecoder` may be we can use it